features/step_definitions/cms/section_steps.rb

```diff
@@ -25,12 +34,12 @@
   provider.sections.create!(options)
 end
 
-Given /^the (section "[^\"]*" of provider "[^\"]*") is access restricted$/ do |section|
+Given "the {section_of_provider} is access restricted" do |section|
```

Comment:
same here. Maybe `section_x_of_provider_y`?

Reply:
we have multiples of this, I think it can be used as a convention the way you have it already